sites/friday/src/manage-resources/_index.md

```diff
@@ -0,0 +1,118 @@
+  - "disk storage"
+  - "horizontal scaling"
+  - "scaling"
+---
+
+{{% vendor/name %}} allows you to configure project resources and adjust them at any time.
+On each of your environments, you can define which amount of CPU, RAM,
+and disk storage you want to allocate to each app and service.
+
+For example, you might want to allocate more resources to your production and staging environments
+than to your development environments.
+This flexibility in resource allocation allows you to optimize performance and costs.
+
+If your app shows signs of struggling with high load, or if you're expecting a traffic spike,
+you can also add instances to your application container.
+Such horizontal scaling allows your app to keep running smoothly,
+and shields your users from performance issues and downtime.
+
+{{< note >}}
+
+When you first deploy your project on {{% vendor/name %}},
+you get notified that you need to configure resources for your project.
+
+This is because {{% vendor/name %}} doesn't know the exact amount of resources your project needs to run smoothly.
+Therefore, your app can only be successfully deployed once you've [configured those resources](#configure-individual-container-resources) through the {{% vendor/name %}} Console or [CLI](/administration/cli/_index.md).
+
+For better guidance on how to configure resources when you first deploy a project,
+make sure you use `{{% vendor/cli %}} push` instead of `git push`.
+
+{{< /note >}}
+
+## Configure individual container resources
+
+{{< codetabs >}}
+
++++
+title= Using the CLI
++++
+
+Run the `resources: set` command, and follow the prompts to set resources for each of your apps and services.
+
+<--->
+
++++
+title= From the Console
++++
+
+1. Navigate to your environment.
+2. Select an app or service in the tree on the left-hand side:
+   ![Apps and services tree](/images/flexible-resources/apps-services-tree.png "0.2")
+   A window pops up for you to configure resources on the current environment:
+   ![Configure your resources on the current environment window](/images/flexible-resources/configure-flexible-resources.png "0.5")
+   The values available in this window depend on the [container profile](#1-configure-your-container-profiles) of each instance.
+3. For each app and service displayed, select a CPU & RAM combination and enter the amount of disk/storage you want to allocate to each container.
+4. For each of your apps, select the number of instances you want to deploy.
+5. Click **Save**.</br>
+   You environment is redeployed.
+
+{{< /codetabs >}}
+
+{{< note title="Tip" >}}
+
+You can keep an eye on your costs in the {{% vendor/name %}} Console:
+
+- To view a monthly estimate of how much each of your projects is expected to cost,
+navigate to your organization and open the **Overview** tab.
+
+- To view a monthly estimate detailing the resources allocated to each instance in a project and the related costs,
+navigate to your project and open the **Overview** tab.
```

Comment:
I'm confused by the location of the "Overview" tab for these two. I can find Overview under a particular environment, but neither under a Project nor an Organization.